data/pages-content/thankyou.md

```diff
@@ -1,368 +1,378 @@
 
+- sangarshanan
 - Vaibhav Srivastav (VB)
 - Diego Russo
-- Sangarshanan
-- Artur Czepiel
-- Laís Carvalho
-- Vicky Twomey-Lee
-- Yasmin Morshed
+- Nicholas Tollervey
 
-And the small and big design stuff you saw including t-shirts, badges, banners,
-stickers and social cards were scrambled together with love by our Design team:
 
-- Laís Carvalho
+## Design
+
+### Conference Designers
+
+Our logo, website assets, t-shirts, banners, badges, lanyards, and general aesthetics of EuroPython 2023 were created by our designers
+- [Anastasia Santoro](https://www.linkedin.com/in/anastasia-santoro-ui-ux-designer-620316157/)
+- [Orlando Festa](https://www.linkedin.com/in/orlandofesta/)
+<!-- - FIXME - ask Patrick if we should put them down below in the Acknowledgement.  -->
+
+### Design Team
+And the rest of the assets, stickers and social cards were scrambled together with love by our Design team.
 - Patrick Arminio (our front-end wizard)
 - Raquel Dou
 
 ## Financial Aid team
 
-If you are one of the 94 people who participated EuroPython 2022 either in
-Dublin or remotely, you have surely exchanged many emails with some of our
-Finaid team members. From spreadsheet wizardry, to responding to countless
-emails and processing all the receipts, here are our kind and organised humans
-behind the Finaid team:
+If you are one of the 107 people who participated in EuroPython 2023 in Prague or remotely, you have surely exchanged many emails with some of our Financial Aid team members. From spreadsheet wizardry, to responding to countless
+emails and processing all the receipts, here are our kind and organised humans behind the Finaid team:
 
 - Sebastiaan Zeff
-- Cheuk Ting Ho
 - Vicky Twomey-Lee
-- Johanna B. Lunghini
-
-## Infrastructure Team
-
-From mailing to conferencing tooling and website, we undertook a big
-infrastructure overhaul this year thanks to our amazing Infrastructure team.
-These humans quietly deploy and manage our services, and are perhaps often
-overlooked until things break:
-
-- Artur Czepiel
-- Silvia Uberti
-- Patrick Arminio
-- Francesco Pierfederici
+- Cheuk Ting Ho
+- Mannie Young
+- Manoj Pandey
 
 ## Programme Team
 
 The folks managing Call for Proposals (CFP), community voting and the entire
 programme selection & scheduling are:
 
-- Alexander CS Hendorf
-- Vaibhav Srivastav (VB)
 - Artur Czepiel
-- Cheuk Ting Ho
+- Vaibhav (VB) Srivastav
+- Cyril Bitterich
+- Rodrigo Girão Serrão
 - Diego Russo
-- Ernesto Arbitrio
-- Nick Radcliffe
-- Patrick Arminio
+- Nicholas Tollervey
+- Cristián Maureira-Fredes
+
+### [Speaker Mentorship Programme](/mentorship)
+
 - Theofanis Petkos
+- Diego Russo
+<!-- - FIXME - Ask Fanis if we are missing people. -->
+
+### Programme Reviewers
 
 The talk selection process was supported by our wonderful reviewers:
 
-- Alexander CS Hendorf
-- Alisa Dammer
-- Andy Piper
-- Anmol Krishan Sachdeva
-- Aurélie Vancraeyenest
+- Agata Migalska
+- Alexander Hess
+- Alisa
```

Comment:
Just noticed that Alisa is missing the family name. Probably it should be `Alisa Dammer`

Reply:
added in https://github.com/EuroPython/website/pull/396/commits/4a5272cda88485783a4c716feb278b4c65ed296b :)